website/docs/maintenance/observability/monitor-metrics.md

```diff
@@ -19,797 +19,297 @@ sidebar_position: 3
-    <thead>
-    <tr>
-      <th class="text-left" style={{width: '225pt'}}>Metrics Name</th>
-      <th class="text-left" style={{width: '165pt'}}>Level</th>
-      <th class="text-left" style={{width: '300pt'}}>Description</th>
-      <th class="text-left" style={{width: '70pt'}}>Type</th>
-    </tr>
-    </thead>
-    <tbody>
-        <tr>
-            <td>numBytesOut</td>
-            <td>Table</td>
-            <td>The total number of output bytes.</td>
-            <td>Counter</td>
-        </tr>
-        <tr>
-            <td>numBytesOutPerSecond</td>
-            <td>Table</td>
-            <td>The output bytes per second.</td>
-            <td>Meter</td>
-        </tr>
-        <tr>
-            <td>numRecordsOut</td>
-            <td>Table</td>
-            <td>The total number of output records.</td>
-            <td>Counter</td>
-        </tr>
-        <tr>
-            <td>numRecordsOutPerSecond</td>
-            <td>Table</td>
-            <td>The output records per second.</td>
-            <td>Meter</td>
-        </tr>  
-    </tbody>
-</table>
\ No newline at end of file
+The identifier is delimited by `metrics.scope.delimiter`. Currently, the `metrics.scope.delimiter` is not configurable, it determined by the metric reporter. Take prometheus as example, the scope will delimited by `_`, so the scope like `A_B_C`, while Fluss metrics will always begin with `fluss`, as `fluss_A_B_C`.
+
+The key/value pairs are called variables and are used to filter metrics. There are no restrictions on the number of order of variables. Variables are case-sensitive.
+```text
+fluss_<scope>[_<infix>]<delimiter><metric-name>
+```
+
+
+1. Begins with `fluss_`.
+2. **Scope** indicates *where* the metric originates (`coordinator`, `tabletserver`, `client`, …).
+3. **Infix** (optional) groups related metrics (e.g., `status_JVM_CPU`).
+4. `<delimiter>` depends on the reporter. For example Prometheus uses `_`.
+5. Finally, the **metric name**.
+
+**Example**
+
+```text
+fluss_tabletserver_status_JVM_CPU_load
+```
+
+
+represents the `recent JVM CPU load` of a `TabletServer`.
+
+---
+
+## Metric Reporters
+
+Fluss ships reporters for Prometheus, JMX and more.  
+See **[Metric Reporters](metric-reporters.md)** for setup details.
+
+---
+
+## Reference
+
+### Cluster-level Metrics 
+#### Scope: Coordinator scope
+
+| Metric                | Description                                  | Type  |
+|-----------------------|----------------------------------------------|-------|
+| `activeCoordinatorCount` | Active CoordinatorServers in the cluster. | Gauge |
+| `activeTabletServerCount` | Active TabletServers.                    | Gauge |
+| `offlineBucketCount`  | Buckets currently offline.                   | Gauge |
+| `tableCount`          | Total tables.                                | Gauge |
+| `bucketCount`         | Total buckets.                               | Gauge |
+| `replicasToDeleteCount` | Replicas scheduled for deletion.           | Gauge |
```

Comment:
The table formatting is causing a Markdown checkstyle failure. Should we adjust it to be aligned?